PubHub/PubHub.AdminPortal/Components/Pages/Modals/Book/UpdateBookModal.razor

```diff
@@ -108,15 +108,14 @@
     public EventCallback<bool> OnClose { get; set; }
 
     [Parameter]
-    public BookInfoModel BookInfoModel { get; set; }
+    public required BookInfoModel BookInfoModel { get; set; }
 
     private BookForm bookUpdateform = new();
-    private IReadOnlyList<ContentTypeInfoModel> contentTypes;
+    private IReadOnlyList<ContentTypeInfoModel>? contentTypes;
 
-    private IReadOnlyList<AuthorInfoModel> authors;
+    private IReadOnlyList<AuthorInfoModel>? authors;
```

Comment:
I think we should do `private IReadOnlyList<AuthorInfoModel> authors = [];` instead of making it nullable and then also remove the `!` at line 75.

Same for `contentTypes` and `!` at line 27 🙂